lib/fourflusher/find.rb

```diff
@@ -72,12 +61,11 @@ def usable_simulators(filter = nil, os = :ios, minimum_version = '1.0')
       os_version = ''
       sims = []
 
-      list(['devices']).lines.each do |line|
-        @os_regex.match(line) do |m|
-          os_name = m['os_name']
-          os_version = m['os_version']
+      JSON.parse(list(['-j', 'devices']))['devices'].each do |runtime_str, devices|
```

Comment:
It just occurred to me what you meant by this. Not sure what I was thinking before. This is fixed now as well.